src/types/category.ts

```diff
@@ -9,3 +10,24 @@ export class CategoryType implements Partial<Category> {
   @Field(_type => String, { nullable: true })
   name?: string;
 }
+
+@InputType()
+export class CategoryInputType {
```

Comment:
혹시 object type 검색에 사용되는 부분들은 네이밍을 다 바꿔주실 수 있으신가요
```suggestion
export class CategoryOptions {
```

Reply:
수정하였습니다